server/server_test.go

```diff
@@ -120,16 +120,13 @@ func (cli *testServerClient) runTests(t *testing.T, overrider configOverrider, t
 
 // runTestsOnNewDB runs tests using a specified database which will be created before the test and destroyed after the test.
 func (cli *testServerClient) runTestsOnNewDB(t *testing.T, overrider configOverrider, dbName string, tests ...func(dbt *testkit.DBTestKit)) {
-	dsn := cli.getDSN(overrider, func(config *mysql.Config) {
-		config.DBName = ""
-	})
+	dsn := cli.getDSN(overrider)
 	db, err := sql.Open("mysql", dsn)
 	require.NoErrorf(t, err, "Error connecting")
```

Comment:
Maybe change the error to "Error in sql.Open()", since it does not actually connect?
I will do connect for the first statement (or Ping() or anything else that needs information from the server).

Reply:
require.NoErrorf(t, err, "Error connecting")  =>  require.NoError(t, err) ?

Reply:
Done